RealmTasks Android/app/src/main/java/io/realm/realmtasks/SignInActivity.java

```diff
@@ -89,6 +92,9 @@ public void onClick(View view) {
 
     private void loginComplete(User user) {
         UserManager.setActiveUser(user);
+
+        createInitialDataIfNeeded();
```

Comment:
What if the user choose to register first time instead, shouldn't we be calling `createInitialDataIfNeeded` ?